app/subscriber/src/features/utils/navigateAndScroll.ts

```diff
@@ -0,0 +1,7 @@
+export const navigateAndScroll = (navigate: any, path: string) => {
```

Comment:
Change this to a hook `useNavigateAndScroll` so that we don't need to pass the `navigate` function.

Also try to never use `any`.  We only use it when we have to.  Once you convert this to a hook you won't need this parameter anyway.